contrib/postgresql-common/postgresql-common.trigger

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```

Comment:
this is also gross and not usable

if you're just taking packaging from alpine here, please don't do that